contracts/contracts/MACI.sol

```diff
@@ -25,7 +25,7 @@ contract MACI is IMACI, DomainObjs, Params, Utilities {
   uint8 public immutable stateTreeDepth;
 
   uint8 internal constant TREE_ARITY = 2;
-  uint8 internal constant MESSAGE_TREE_ARITY = 5;
+  uint8 internal constant MESSAGE_BATCH_SIZE = 20;
```

Comment:
let's remove this, it should be a parameter to the `deployPoll` function

Reply:
We might encounter an issue with the number of parameters for deployPoll, similar to what we had with changes from milestone 2. Let's discuss a general solution for parameter organization.

Reply:
Is that Stack too deep error?